development/build/index.js

```diff
@@ -117,4 +133,77 @@ function defineAllTasks() {
 
   // special build for minimal CI testing
   createTask('styles', styleTasks.prod);
+
+  // Finally, start the build process by running the entry task.
+  runTask(entryTask, { skipStats });
+}
+
+function parseArgv() {
+  const NamedArgs = {
+    BetaVersion: 'beta-version',
+    BuildType: 'build-type',
+    OmitLockdown: 'omit-lockdown',
+    SkipStats: 'skip-stats',
+  };
+
+  const entryTask = process.argv[2];
```

Comment:
does minimist provide positional args parsing we can use instead

Reply:
Not as far as I can tell: https://www.npmjs.com/package/minimist

Reply:
its the underbar, eh `{ _: [ 'foo', 'bar', 'baz' ], `

Reply:
Yeah! Which seems, roughly equivalent to what I'm doing?

Hm, actually. I can use the underbar to make this validation stricter. Hold on.

Reply:
2ff827c